### ISSUE TITLE
Add support for Windows systems.

### ISSUE THREAD

**Issue by vilihegyi** [vilihegyi/Logger#1]
Currently the logger only works for UNIX based systems. For a better user experience, Windows systems should have the support added too.

**Comment by svenkat19** [comment]
I would like to help with Windows computers.

**Comment by vilihegyi** [comment]
Sure, take it on yourself, make a pull request once you're done and I'll review it. Thanks for considering it!

**Comment by svenkat19** [comment]
Can u give me an idea as to where I can start from  ?

**Comment by vilihegyi** [comment]
I'd suggest trying to make a Windows version, maybe in Visual Studio and see if it works like in Linux. Basically what I'd like to see is a structure as follows when doing debugging:
Current time (hours, mins, seconds) followed by debug level (DEBUG, INFO, WARNING, ERROR, FATAL), followed by the class name and line where the log was used, and followed by the debug message.

I'll attach a print screen to show how I'd like it to be written code-wise and what output to expect:

![image](https://user-images.githubusercontent.com/22892306/126077293-9fce0771-75d0-4bfb-885e-73fd548fcbb7.png)

**Comment by svenkat19** [comment]
It works fine on a Windows PC
![image](https://user-images.githubusercontent.com/74498344/126158180-7d3fa7fa-1083-42f3-a289-0c8ccaecb9d4.png)

**Comment by vilihegyi** [comment]
Judging from the screenshot, I guess you are using VS Code. I specified Visual Studio (I tested it on that and it failed, so it's guaranteed not to work on that). in the comment. I'll be a bit more clear this time: Visual Studio (2019 - Community Edition) doesn't compile and run this, so we should have support for the VS compiler too.

**Comment by svenkat19** [comment]
![image](https://user-images.githubusercontent.com/74498344/126163500-17775a0a-7bf9-4acf-8c1b-bfc165468a0e.png)

**Comment by vilihegyi** [comment]
Here's what I am getting when using Visual Studio 2019:
![image](https://user-images.githubusercontent.com/22892306/126368700-b27a7f5e-48de-4d96-a3ec-f13f7377da15.png)

This is the result for (_MSC_VER) version 1928 when running the above code.

**Comment by svenkat19** [comment]
Could you try uninstalling VS and downloading it again?

**Comment by vilihegyi** [comment]
> 
> 
> Could you try uninstalling VS and downloading it again?

I kindly hope this is not an official suggestion you propose. Also, I did update Visual Studio (I'm not going to uninstall 80 GB and reinstall it just to "Seem to fix the issue") and it is still NOT working. I believe our compilers differ and that's why in your case it may work and mine not. (or you had added the option to disable certain warnings, which I do NOT recommend!)

Anyways I found a fix, to check platform specifically if we use Windows or not and I'll add the required changes in the next commit.
